core/src/main/scala/org/apache/spark/sql/rapids/tool/util/RuntimeUtil.scala

```diff
@@ -73,6 +82,30 @@ object RuntimeUtil extends Logging {
       "os.version" -> System.getProperty("os.version")
     )
   }
-}
 
+  def getJVMHeapArguments: Map[String, String] = {
+    val gcMxBeans = ManagementFactory.getGarbageCollectorMXBeans
```

Comment:
Wanted to check if it was used for debugging? I don't see it being used else where in the function below.